docs/getting-started-guides/coreos/coreos_multinode_cluster.md

```diff
@@ -59,9 +59,9 @@ aws ec2 run-instances \
 --user-data file://node.yaml
 ```
 
-### GCE
+### Google Compute Engine
```

Comment:
Is it worth adding "(GCE)" here and in `coreos_single_node_cluster.md` ? I see you added it in a bunch of files below. I trust your judgement, just checking.